nvflare/private/fed/server/server_json_config.py

```diff
@@ -66,7 +66,6 @@ def __init__(self, config_file_name: str, args, app_root: str, kv_list=None, exc
             SystemVarName.JOB_ID: args.job_id,
             SystemVarName.SITE_NAME: "server",
             SystemVarName.WORKSPACE: args.workspace,
-            SystemVarName.ROOT_URL: args.root_url,
             SystemVarName.SECURE_MODE: self.cmd_vars.get("secure_train", True),
```

Comment:
Since we DO NOT pass the root_url as arg when starting server, I remove it here.

Or do we WANT to pass the root_url when server is starting?

Reply:
I guess this is fine for now since we don't have a case of using CellPipe on the server side.